xarray/core/utils.py

```diff
@@ -266,13 +266,15 @@ def either_dict_or_kwargs(
         return cast(Mapping[Hashable, T], kw_kwargs)
 
 
-def is_scalar(value: Any) -> bool:
+def is_scalar(value: Any, test0d: Optional[bool] = True) -> bool:
```

Comment:
```suggestion
def is_scalar(value: Any, include_0d: bool = True) -> bool:
```

Reply:
Yes, the `Optional` type is for indicating values that can be `None`, which isn't the case here.

Reply:
Thanks for the correction. It's done.